### PR TITLE
Judging view container

### LINKED ISSUE #182: /judging/view container
title

- fetch data from backend
- submit form

### PR DESCRIPTION
**Pull request by ianmah**
closes #182

making todo issues for the rest of this flow 
ezgif com-gif-maker

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -151,9 +163,7 @@ function App() {
           <AdminAuthPageRoute path="/judging/admin">
             <JudgingAdmin />
           </AdminAuthPageRoute>
-          <AuthPageRoute path="/judging/view/:id">
-            {params => <JudgingView id={params.id} />}
-          </AuthPageRoute>
+          <Route path="/judging/view/:id" component={JudgingViewContainer} />
```
Comment (ianmah) [inline src/App.js:166]: because we need `params` to be accessible directly under the `<Route>` component

File: src/components/Input/TextArea.js [hunk 2]
```diff
@@ -52,11 +48,14 @@ export const TextArea = ({
         }`}
         {...rest}
       />
-      {invalid && <ErrorMsg> {errorMsg} </ErrorMsg>}
+      {invalid && <ErrorMessage> {ErrorMessage} </ErrorMessage>}
```
Comment (ianmah) [inline src/components/Input/TextArea.js:51]: ah crap one sec

File: src/App.js [hunk 3]
```diff
@@ -98,6 +99,17 @@ const ApplicationFormContainer = ({ params }) => {
...
+  return isAuthed ? (
+    <Page>
+      <JudgingView id={params.id} />
+    </Page>
+  ) : (
+    <Redirect to="/login" />
+  )
+}
+
 function App() {
```
Comment (jackyzha0) [inline src/App.js:113]: why is this needed? doesn't `AuthPageRoute` already handle this redirect stuff

**Review by jackyzha0 — APPROVED** [review]
small question about AuthPageRoute but rest looks goood
